pkg/kp/rollstore/consul_store.go

```diff
@@ -392,11 +392,7 @@ func (s consulStore) CreateRollingUpdateFromOneMaybeExistingWithLabelSelector(
 
 	var oldRCID rc_fields.ID
 	if len(matches) > 1 {
-		return roll_fields.Update{}, util.Errorf(
-			"Can't create update: old RC selector %s was ambiguous, provided %d matches",
-			oldRCSelector.String(),
-			len(matches),
-		)
+		return roll_fields.Update{}, AmbiguousRCSelector
```

Comment:
I could implement an error type and return it, but that opens a different can of worms re: nil checking but maybe worth it?